packages/beacon-node/test/spec/presets/fork_choice.test.ts

```diff
@@ -112,6 +112,9 @@ const forkChoiceTest =
           }
         );
 
+        // The handler of `ChainEvent.forkChoiceFinalized` access `db.block` and raise error if not found.
+        chain.emitter.removeAllListeners(ChainEvent.forkChoiceFinalized);
```

Comment:
This handler is throwing an error `block not found`, which is `unhandled rejection` even in previous implementation but never been caught by mocha.